extras/task_yaml_path_checker.py

```diff
@@ -0,0 +1,93 @@
+from typing import Dict
+
+import neurobooth_os.iout.metadator as meta
+from neurobooth_os.iout.stim_param_reader import StudyArgs, CollectionArgs, RawTaskParams, InstructionArgs, \
+    StimulusArgs, DeviceArgs, SensorArgs
+
+
+class PathError(Exception):
+    pass
+
+
+def check_paths_from_study(study_name: str):
+    """
+    Reads and parses all yaml files found in NB_CONFIG folder. The named study is checked to
+    ensure that every element referenced from that study has an associated yaml file.
+
+    Parameters
+    ----------
+    study_name Name of the study to check
+
+    Returns
+    -------
+
+    """
+    print("Note: Checking against yaml files in location specified in the NB_CONFIG environment variable.")
+    studies: Dict[str, StudyArgs] = meta.read_studies()
+    if study_name not in studies:
+        raise PathError(f"No yaml file was found for study '{study_name}' ")
+    else:
```

Comment:
Could just remove this else and save indentation below since the if branch raises an error. Same comment applies multiple times below.